lib/fhir.js

```diff
@@ -18,25 +18,30 @@ function request (options) {
   }
 
   const client = axios.create({
-    baseURL: `${config.get(`${environment}.fhirUrl`)}`
+    baseURL: `${config.get(`${environment}.fhirUrl`)}`,
+    headers: {
+      'Prefer': 'handling=strict'
+    }
   });
   client.interceptors.request.use(tokenProvider);
   axiosRetry(client, { retries: 3 });
   return client;
 }
 
-module.exports.get = function (options, path) {
-  return request(options).get(path);
+module.exports.request = request;
+
```

Comment:
I noticed one of the FHIR commands was passing a config object to the post function, but the post function was ignoring it. It worked regardless for some reason, but it seemed like an accident waiting to happen to not take a config here like axios does.